docs/_locale/_pot/api.pot

```diff
@@ -394,7 +394,7 @@ msgid "Current value of the 'Content-Length' header."
 msgstr ""
 
 #: ../../../bottle.py:docstring of bottle.FileUpload.get_header:1
-msgid "Return the value of a header within the mulripart part."
+msgid "Return the value of a header within the multipart part."
```

Comment:
I usually avoid touching localizations/pot files, but for this project, I decided to include the changes. It's trivial for me to drop changes to files, I just need to know which...